local/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	hookdeck "github.com/hookdeck/hookdeck-go-sdk"
+	hookdeckclient "github.com/hookdeck/hookdeck-go-sdk/client"
+)
+
+func main() {
+	if err := run(); err != nil {
+		os.Stderr.WriteString(err.Error())
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	client := hookdeckclient.NewClient()
```

Comment:
Can we create a new client with some proper config so we can run it locally with `go run main.go`? This is what I have locally but the API key obviously needs to come from somewhere else

```go
client := hookdeckClient.NewClient(
	hookdeckClient.ClientWithBaseURL("http://localhost:9000/2023-07-01"),
	hookdeckClient.ClientWithAuthToken("31gbmrt1bz4917d2ikoo5tjf1u7hepvpdkfn85ad5ttelefexc"),
)
```

Reply:
@alexluong feel free to modify the client however you want -- this was just setup as an example

Reply:
sent this in the main comment, but yes i can look pick this work up further if that's easier

> I'm also happy to take it from here too if you prefer. Mainly I don't know how to continue working on this on your fork but I can look into it or open a new PR with my own branch.